io.github.twisterarmy.Twister.json

```diff
@@ -0,0 +1,94 @@
+{
+    "app-id": "io.github.twisterarmy.Twister",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
```

Comment:
current is 24.08

Reply:
well, you should figure out how to solve the problem. 23.08 will be EOL in about 4 month. We won't accept a submission using it.